test/cmd/migrate.sh

```diff
@@ -78,4 +78,14 @@ os::cmd::expect_success_and_not_text 'oc get is test --template "{{ range .statu
 os::cmd::expect_success_and_not_text 'oc get is test --template "{{ range .status.tags }}{{ range .items }}{{ .dockerImageReference }}{{ \"\n\" }}{{ end }}{{ end }}"' '^mysql'
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "cmd/migrate/legacyhpa"
+# create a legacy and a normal HPA
+os::cmd::expect_success 'oc create -f test/testdata/hpa/legacy-and-normal-hpa.yaml'
+# verify dry run
+os::cmd::expect_success_and_text 'oc adm migrate legacy-hpa' 'migrated=1'
```

Comment:
I assume you also need to run once with --confirm?

Reply:
yep, whoops